server/flaskr/database/postgres/tenant_context.py

```
@@ -0,0 +1,19 @@
+from contextvars import ContextVar
+from contextlib import contextmanager
+from typing import Optional
+
+_tenant_org_id: ContextVar[Optional[str]] = ContextVar("tenant_slug", default=None)
+
+@contextmanager
+def use_tenant(id: Optional[str]):
```

Ruff (0.15.15)

[error] Function argument `id` is shadowing a Python builtin

(A002)